examples/tfengine/generated/team/devops/terraform.tfvars

```diff
@@ -29,7 +35,17 @@ project = {
     "secretmanager.googleapis.com",
   ]
   owners_group = {
-    id = "example-devops-owners@example.com"
+    customer_id  = ""
+    description  = ""
+    display_name = "example-devops-owners"
+    exists       = true
+    id           = "example-devops-owners@example.com"
+    owners = [
```

Comment:
Can this be formatted as `owners = []` ?

Reply:
I mentioned here https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/984#discussion_r682769050

But I think that's preferrable. Other functions like hcl write the emails without the "

```
owners = [user@example.com]
```

instead of

```
owners = ["user@example.com"]
```

I didn't found other way around

Reply:
I actually mean removing the line break if it's an empty list, 

like [], instead of [
]


but it's very minor.

Reply:
Resolved in favor of https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/984#:~:text=I%20mentioned%20here-,%23984%20(comment),-But%20I%20think